addons/hr_expense/models/hr_expense.py

```diff
@@ -347,13 +347,14 @@ def create_expense_from_attachments(self, attachment_ids=None, view_type='list')
             product = product.filtered(lambda p: p.default_code == "EXP_GEN")[:1] or product[0]
         else:
             raise UserError(_("You need to have at least one category that can be expensed in your database to proceed!"))
+        default_category = self.env.company.expense_category_id
```

Comment:
I was gonna ask you to rename this variable `default_expense_category` but I think it's actually better to just get rid of it. I don't think it adds much more readability.